CRC.h

```diff
@@ -0,0 +1,1699 @@
+/**
```

Comment:
Also add the license file from the [git repo](https://github.com/d-bahr/CRCpp) to the LICENSES subdirectory for binary redistribution.